_posts/2022-06-21-ejb-over-tls.adoc

```diff
@@ -0,0 +1,343 @@
+---
+layout: post
+title: "Securing Jakarta Enterprise Beans with mutual TLS authentication"
+date: 2022-06-21
+tags: ejb tls sasl https credential-store keystore truststore
+synopsis: A demonstration on how to configure Elytron for secure connections to EJBs, using mutual TLS, SASL authentication mechanisms, and credential stores.
+author: cam-rod
+---
+
+:toc: macro
```

Comment:
Thanks for the PR, @cam-rod! We actually don't need to duplicate the full post here. We just need to include the necessary metadata to link to the blog post on the Elytron website. This PR has some examples of how to do that:

https://github.com/wildfly/wildfly.org/pull/327

Notice that the `link` tag is included in the examples above to just link directly to the post.